static/js/textMarkIconsPosition.js

```diff
@@ -25,25 +25,26 @@ textMarkIconsPosition.prototype.updateIconsPosition = function() {
   // hide text mark icons while their position is being updated
   this.hideIcons();
   var self = this;
+  var $lines = utils.getPadInner().find('div');
 
-  var inlineTextMarks = this._getTextMarkIdAndItsPosition();
+  var inlineTextMarks = this._getTextMarkIdAndItsPosition($lines);
```

Comment:
Reparei que vc mudou a assinatura de várias funções, para passar o `$lines` ao invés de usar o `utils.getPadInner()`. Como o padInner tem uma estrutura pouco complexa, e as linhas são todas filhas do `body`, não acho que isso mude tanto a performance assim. Se der, faz um teste usando `$lines` e usando `utils.getPadInner()` no lugar, para ver se muda muito. Se for pouca coisa, eu diria para ficarmos com o código mais simples